src/gui/widgets/chatbox.cpp

```diff
@@ -89,14 +89,25 @@ void chatbox::finalize_setup()
 
 void chatbox::load_log(std::map<std::string, chatroom_log>& log, bool show_lobby)
 {
+	const std::string new_tip = formatter() << markup::span_color("#FF0000", "\n============NEW============");
```

Comment:
translatable

Reply:
I'd prefer to have only the 'NEW' marked for translation and not the whole string.

Also, just wondering if a hyphen or dash would be look nicer instead of the equals sign.

Reply:
I think `NEW` becomes `" + _("NEW") + "`, and it's probably helpful to add something like `# po: New chat text indicator immediately above it`.

Reply:
po files are not modified directly by developers (or so I understand). See other comments.